Gems/ScriptEvents/Code/Include/ScriptEvents/ScriptEventDefinition.h

```diff
@@ -151,7 +151,7 @@ namespace ScriptEvents
         
         const AZStd::vector<Method>& GetMethods() const { return m_methods; }
 
-        AZStd::string_view GetLabel() const { return GetName(); }
+        AZStd::string_view GetLabel() const { return { m_name.Get<AZStd::string>()->data(), m_name.Get<AZStd::string>()->size() }; }
```

Comment:
Actually it would be better to change the return type to `AZStd::string` That way the Return Value Optimization can take place